### PR TITLE
Various documentation improvements.

### PR DESCRIPTION
**Pull request by Hixie**
For example, mention the icon used for the drawer menu in the docs (this helps people writing unit tests); add DefaultAssetBundle sample code.

### CODE REVIEW

**Comment by Hixie** [comment]
cc @gspencergoog

File: dev/bots/analyze-sample-code.dart [hunk 1]
```diff
@@ -8,6 +8,8 @@ import 'dart:io';
 
 import 'package:path/path.dart' as path;
 
+/// To run this: bin/cache/dart-sdk/bin/dart dev/bots/analyze-sample-code.dart
```
Comment (gspencergoog) [inline dev/bots/analyze-sample-code.dart]: Shouldn't that just be '//'?

**Review by gspencergoog — APPROVED** [review]

**Comment by gspencergoog** [comment]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

File: dev/bots/analyze-sample-code.dart (hunk 1 above)
Comment (Hixie) [reply dev/bots/analyze-sample-code.dart]: fixed

**Comment by Hixie** [comment]
thanks for the review. will land on green.

**Pull request merged**